slang_frontend.cc

```diff
@@ -1008,28 +1005,10 @@ struct ProceduralVisitor : public ast::ASTVisitor<ProceduralVisitor, true, false
 			if (!cv.as_const().as_bool())
 				break;
 
-			RTLIL::SigSpec disable;
-
-			if (disableLoop_outs.empty()) {
-				disable = RTLIL::S0;
-			} else {
-				disable = disableLoop_outs.back();
-			}
-
-			// add next disable wire
-			RTLIL::Wire *breakNextLoop = netlist.canvas->addWire(NEW_ID_SUFFIX("breakNextLoop"), 1);
-			RTLIL::SigSpec breakNextLoopOred = netlist.LogicOr(breakNextLoop, disable);
-			do_simple_assign(stmt.sourceRange.start(), breakNextLoop, RTLIL::S0, false);
-
-			disableLoop_ins.push_back(breakNextLoop);
-			disableLoop_outs.push_back(breakNextLoopOred);
-
-			eval.push_frame();
```

Comment:
We lost the `push_frame()`/`pop_frame()` calls per iteration, these are important if there are variables local to the loop body

Reply:
Yep, fixed!